Config.sample.toml

```diff
@@ -0,0 +1,276 @@
+# Set WEBHOOK_ID and WEBHOOK_TOKEN as explained here https://hackaday.com/2018/02/15/creating-a-discord-webhook-in-python/
+# Set USER_ID as explained here https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID-
+# === WEBHOOK_ID and WEBHOOK_TOKEN ===
+# Follow the steps here: https://hackaday.com/2018/02/15/creating-a-discord-webhook-in-python/
+WEBHOOK_ID = ""
+WEBHOOK_TOKEN = ""
+# === USER_ID ===
+# This is your Discord User ID for mentioning you.
+# More info: https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID-
+USER_ID = ""
+
+# ==========
+# === LANGUAGE SETTINGS
+[language]
+# === LANGUAGE ===
+# Set the language of your game here.
+# Currently we support English, Spanish, French, Korean, and German
+LANGUAGE = "English"
+
+# ==========
+# === POKEMON DATA SETTINGS
+# These paths are if you are not using the pre-calculated values included with the program
+# and should not be modfied unless you know what you are doing.
+[pokemon_data_paths]
+Boss_Pokemon = "data/boss_pokemon.json"
+Rental_Pokemon = "data/rental_pokemon.json"
+Boss_Matchup_LUT = "data/boss_matchup_LUT.json"
+Rental_Matchup_LUT = "data/rental_matchup_LUT.json"
+Rental_Pokemon_Scores = "data/rental_pokemon_scores.json"
+path_tree_path = "data/path_tree.json"
+type_icon_path = "data/type_icons.pickle"
+
+# ==========
+# === OTHER LANGUAGE SETTINGS
+# What follows are how to set the various phrases that allow
+# the program to detect what is currently happening in your game.
+# These may need to be modified if your capture card adds extra noise,
+# modifies your colors, or other weird things. Feel free to reach out to
+# a developer if you're having issues.
+[English]
+TESSERACT_LANG_NAME = "eng"
+DATA_LANG_NAME = "en"
+BACKPACKER = "backpacker"
+SCIENTIST = "swapping"
+PATH = "path"
+FAINT = "The storm"
+START_PHRASE = "Would you like to embark"
+WEATHER_CLEAR = "PLACEHOLDER"
+WEATHER_SUNLIGHT = "PLACEHOLDER"
+WEATHER_RAIN = "PLACEHOLDER"
+WEATHER_SANDSTORM = "PLACEHOLDER"
+WEATHER_HAIL = "PLACEHOLDER"
+TERRAIN_CLEAR = "PLACEHOLDER"
+TERRAIN_ELECTRIC = "PLACEHOLDER"
+TERRAIN_GRASSY = "PLACEHOLDER"
+TERRAIN_MISTY = "PLACEHOLDER"
+TERRAIN_PSYCHIC = "PLACEHOLDER"
+
+[Spanish]
+TESSERACT_LANG_NAME = "spa"
+DATA_LANG_NAME = "sp"
+BACKPACKER = "mis objetos"
+SCIENTIST = "préstamo"
+PATH = "quieres seguir"
+FAINT = "La tormenta"
+START_PHRASE = "emprender"
+WEATHER_CLEAR = "PLACEHOLDER"
+WEATHER_SUNLIGHT = "PLACEHOLDER"
+WEATHER_RAIN = "PLACEHOLDER"
+WEATHER_SANDSTORM = "PLACEHOLDER"
+WEATHER_HAIL = "PLACEHOLDER"
+TERRAIN_CLEAR = "PLACEHOLDER"
+TERRAIN_ELECTRIC = "PLACEHOLDER"
+TERRAIN_GRASSY = "PLACEHOLDER"
+TERRAIN_MISTY = "PLACEHOLDER"
+TERRAIN_PSYCHIC = "PLACEHOLDER"
+
+[French]
+TESSERACT_LANG_NAME = "fra"
+DATA_LANG_NAME = "fr"
+BACKPACKER = "Randonneuse"
+SCIENTIST = "intéresse"
+PATH = "chemin"
```

Comment:
PATH = "Quel chemin"

I changed that in the previous PR

Reply:
```suggestion
PATH = "Quel chemin"
```